### DISCUSSION TITLE
Welcome to LosslessCut Discussions!

### DISCUSSION THREAD

**Discussion by mifi** [mifi/lossless-cut#534] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by neostevenyan** [comment]
I think I have found you! Most wanted needs really have been understood by this programme, cheers！

**Comment by MMMarcis** [comment]
Thank you for such an awesome tool! It's been a time saver!

1. Why does it take 15 seconds to start the tool on a modern PC with SSD?

2. Could compression be incorporated in this? I understand this is quite contradictory but such a feature could make this the best lightweight editing software!
3. Would cropping be possible?

**Reply by mifi** [reply]
1. what's your specs? and os?
2. I might add some basic accure cutting with re-compression. See #372
3. I don't think that's gonna happen anytime soon

**Comment by Adil15101** [comment]
Made an account just to star this, it's so good! Just wanted to ask one question. Will you be willing to add a basic transition option to add between the cuts? I know that's not the purpose of this app but it would be a great addition to this wonderful app.

**Reply by mifi** [reply]
Hi! I dont' think transitions will ever happen because it's a lossy operation and quite compless. But check out my other tool [editly](https://github.com/mifi/editly)

It can do many cool transitions. However unfortunately it's currently only for more advanced users (command line / node.js)

**Comment by Shaggy84675** [comment]
Hello,

I really love the tool. It's so easy to use and it has so many features. I have one question. Would be possible to implement merge for audio tracks? Like I'm using this tool mostly to cut out my games DVR. And GeForce Experience has an option to separate audio track and mic track. Sometimes it'd be useful to merge these tracks into one track.

Thank you.

**Reply by mifi** [reply]
Hi. mixing audio is a lossy operation so it's not the primary focus of losslesscut, but see this issue #372

**Comment by NoahIles** [comment]
Feature request: Segment Speed, I believe FFmpg allows you to change the speed of a clip, would it be possible to give us a button on each segment that allows us to change the speed, for slomo clips, or speeding through a section of a video instead of cutting. Love the app best in class in my experience so far.

**Reply by mifi** [reply]
Sorry slomo is probably never going to happen. It's possible with my other tool [editly](https://github.com/mifi/editly) however it's mostly for advanced users.

**Comment by Ralhaz** [comment]
Hey mate thats a great cutting program really i LOVE it, Is it possible to "add another" videos and merge them?

**Reply by mifi** [reply]
Not currently possible. You have to do it in separate steps. See #423

**Comment by davidtri1** [comment]
First, thanks for making this software.  When I add subtitles to mkv files, the subtitles are displayed on the left side of the screen.  They display correctly when added to mp4 files.  Because of this, I export everything in mp4.  The default output is always mkv, which means I have to select mp4 every time I save/export a file.  
1.  Please  fix it so that the mkv output file display subtitles correct.
2.  Or please leave the output file default to "last use" 
Thank you so much.

**Comment by mifi** [comment]
> * Please  fix it so that the mkv output file display subtitles correct.

this is most likely a bug in ffmpeg. i don't know how to fix that

> 2\. Or please leave the output file default to "last use"

after selecting mp4, you can click the lock icon next to the format in the top right:
<img width="126" alt="Screenshot 2024-05-03 at 21 39 17" src="https://github.com/mifi/lossless-cut/assets/402547/560bc7a1-8ed4-4d17-b0ce-f7b3feb41cb6">

**Reply by davidtri1** [reply]
Thank you so much for your help.  Locking in the output format saves me a lot time.   As for mkv file displaying srt subs incorrectly, it does not seem to be a problem when I use ffmpeg to do it.  I would rather use Losslesscut to do it though, as there is no typing involved.

Below is are the commands I use.  For mp4 I put -c:s mov_text.

ffmpeg -i "input.mkv" -i "sub1.vtt" -i "sub2.vtt" -c:s srt -c:v copy -c:a copy -map 0 -map 1 -map 2 "output.mkv"

**Reply by mifi** [reply]
does it work if you don't specify `-c:s srt` but instead `-c:s copy`?

I just tested adding a vtt file to a video and then exporting to mkv, and it successfully shows the subtitle. can you share an example vtt file that reproduces the problem?

**Reply by davidtri1** [reply]
-c:s should work.  Thank you.

I added a vtt sub to an mkv file just now.  Still showing sub on the left side of the screen.  Webm and Mp4 work though.
![sub sample](https://github.com/mifi/lossless-cut/assets/168238803/776972da-40a6-4892-8f87-48dbc36b5cd7)

**Reply by mifi** [reply]
unfortunately i don't know how to reproduce this problem

**Reply by davidtri1** [reply]
I have noticed that when I export the file as mkv, the vtt subtitle is copied over to the new file.  It shows "webvtt" sub when I look at it. When I export the file as mp4, the vtt sub become a text-based subtitle.  It shows up as "text" subtitle.

Just to be clear, when exported as mkv, the file displays the subtitle, but on the left side of the screen and is unmovable.

I think -c copy or -c:s copy was used instead of -c:s srt.

**Comment by davidtri1** [comment]
Will ac3 be supported in the future?  It is the audio used in dvd movies.  I hope to, one day, replace VideoRedo with losslesscut.

**Reply by mifi** [reply]
see #364
